### ISSUE TITLE
Accessibility: Linking between date picker input element and its popover

### ISSUE THREAD

**Issue by georgwittberger-t-systems-mms** [nathanreyes/v-calendar#1180]
Current behavior:

Popover of date pickers cannot be linked to input element properly using ARIA attributes. This makes it complicated to tell screen readers that the input element controls visibility of the popover.

Issues are caused by missing attributes on popover wrapper `.vc-popover-content-wrapper` which is populated with calendar pane.

- Wrapper does not have any ID attribute, thus making it impossible to refer to it from the input element using `aria-controls="wrapper-id-here"`.
- Wrapper does not have any ARIA role, so it is unclear which value to use on input element for `aria-haspopup` attribute.

Expected behavior:

Popover of date pickers should be linkable to input element properly using ARIA attributes. This would allow screen readers to announce presence of a related popup.

Further notes:

Possible solution could be setting ARIA attributes on popover wrapper `.vc-popover-content-wrapper` as follows:

- Attribute `role="dialog"` to indicate the popover is a kind of dialog. Using `grid` would be more reasonable for a table-like calendar view but this would require rows and columns to be represented in DOM which is not the case at the moment. VCalendar just renders all days as sibling elements in one big container, then using CSS grid to achieve table styling.
- Attribute `id="value-of-a-new-prop"` to put unique ID on popover wrapper. Value should be given as prop to `v-date-picker`.
- Attribute `aria-hidden="true"` while popover is closed. This is required as soon as ARIA role is changed to something like `dialog`. Otherwise, screen readers may read the wrapper element even when popover is closed.
